Makefile

```diff
@@ -413,6 +413,10 @@ clean-artifacts: ## Remove all dumped artifacts from test runs (see dump_artifac
 push: ## Push Grapl containers to Docker Hub
 	docker-compose --file=docker-compose.build.yml push
 
+.PHONY: push-local
+push-local: ## Push Grapl container to local registry
+	REGISTRY=localhost:5000 docker-compose --file=docker-compose.build.yml push
```

Comment:
actually, we might be able to include this REGISTRY= as part of the WITH_LOCAL_GRAPL_ENV stuff - thoughts? 

That'd be more in line with the rest of our makefile

(also if we could rename it to like DOCKER_REGISTRY or something a bit more namespaced thatd be cool)

Reply:
Ahh! My bad I haven't had to update the makefile before. That's a cool pattern I'll use instead here.